tests/samples.dat

```diff
@@ -1,6 +1,5 @@
 0.0 0.017453293 0.0
 0.0 0.017453293 0.0
-10000
```

Comment:
This line is the primary reason the tests were failing. It seems to be an error so I just deleted it.